test/e2e/nodefeature/nodefeature.go

```diff
@@ -38,7 +38,7 @@ var (
 	DeviceManager = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("DeviceManager"))
 
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
```

Comment:
please fill up the TODO while you are on it

Reply:
Will do in a separate PR. There are the same TODO comments for all the features in this file.